src/App.js

```diff
@@ -13,16 +14,25 @@ const toDoItems = [
 ];
 
 function App() {
+  const [toDoList, setToDoList] = useState([])
+
+  function addToDoList(title) {
+    let newItemArray = {id: "", title, completed: false};
+    let newToDoList = toDoList.slice();
+    newToDoList.push(newItemArray);
+    setToDoList(newToDoList);
```

Comment:
zamiast robic slice,
to mozesz zrobic po prostu setToDoList([...toDoList, newItemArray])
(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax)